### ISSUE TITLE
Error thrown when no dependency is found

### ISSUE THREAD

**Issue by ghost** [BePo65/license-report#126]
Hello. 
I'm using this in order to update lists of dependencies inside a few readmes. If the project has no dependency, it will throw this error. 

```typescript
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at default (file:///usr/local/lib/node_modules/license-report/node_modules/tablemark/dist/index.js:12:25)
    at formatAsMarkdown (file:///usr/local/lib/node_modules/license-report/lib/getFormatter.js:109:9)
    at file:///usr/local/lib/node_modules/license-report/index.js:59:17
```

I would expect it to just show nothing.

**Comment by BePo65** [comment]
Fixed with pr #127.

**Comment by ghost** [comment]
Thank you.
